### ISSUE TITLE
nolint directive is not feature compatible with gometalinter

### ISSUE THREAD

**Issue by alecthomas** [golangci/golangci-lint#65]
In gometalinter, nolint directives can be applied to a branch in the AST, as well as directly on the offending line. golangci-lint does not support the former.

For example, the following does not work:

```go
// nolint
func someFunc() {
  anUnusedVar := true
}
```

while this does work:

```go
func someFunc() {
  anUnusedVar := true // nolint
}
```

It would be very helpful if golangci-lint were compatible with gometalinter for nolint directives.

**Comment by jirfag** [comment]
thank you for the issue, fixed in master and in the latest release v1.5

**Comment by alecthomas** [comment]
Perfect, thanks!
